app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    exist_file, new_name = split_command[1], split_command[2]
+
+    directories = split_command[-1].split("/")
+    new_file = directories[-1]
+
+    src_file = os.getcwd() + "\\" + exist_file
+    last_dir = "\\".join(directories[:-2])
+    parent_dir = os.getcwd() + "\\" + last_dir
+
+    if len(directories) == 1:
+        return os.rename(exist_file, new_name)
+
+    dst_file1 = parent_dir + "\\" + directories[-2] + "\\" + exist_file
+    dst_file2 = parent_dir + "\\" + directories[-2] + "\\" + new_file
+
+    path = os.getcwd()
+    for directorie in directories[:-1]:
+        path += "\\" + directorie
```

Comment:
Your code should be cross-platformed. This will work only on windows. Rewrite it by using neccessary functions from os module, such as os.path.split and os.path.join